otelconnect.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	version             = "0.0.1-dev"
+	version             = "0.6.0-dev"
```

Comment:
After this is merged, would it be possible to do a release of 0.6.0?

Reply:
Sure!